plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/edit.tsx

```diff
@@ -17,50 +17,34 @@ import { ProductGalleryBlockSettings } from './block-settings/index';
 import type { ProductGalleryBlockAttributes } from './types';
 
 const TEMPLATE: InnerBlockTemplate[] = [
+	[ 'woocommerce/product-gallery-thumbnails' ],
 	[
-		'core/group',
-		{
-			layout: {
-				type: 'flex',
-				flexWrap: 'nowrap',
-				verticalAlignment: 'top',
-			},
-			metadata: {
-				name: 'Gallery Area',
-			},
-			className: 'wc-block-product-gallery__gallery-area',
-		},
+		'woocommerce/product-gallery-large-image',
+		{},
 		[
-			[ 'woocommerce/product-gallery-thumbnails' ],
 			[
-				'woocommerce/product-gallery-large-image',
-				{},
-				[
-					[
-						'woocommerce/product-sale-badge',
-						{
-							align: 'right',
-							style: {
-								spacing: {
-									margin: {
-										top: '4px',
-										right: '4px',
-										bottom: '4px',
-										left: '4px',
-									},
-								},
+				'woocommerce/product-sale-badge',
+				{
+					align: 'right',
+					style: {
+						spacing: {
+							margin: {
+								top: '4px',
+								right: '4px',
+								bottom: '4px',
+								left: '4px',
```

Comment:
This is unrelated to this PR but why don't we set the default style in the metadata file?

Reply:
Ah ha! My bad, we indeed shouldn't set default margin for on sale badge block. But what about the gallery navigation bellow?

Reply:
> Ah ha! My bad, we indeed shouldn't set default margin for on sale badge block. But what about the gallery navigation bellow?

Good point!!

Reply:
Ah, but wait. This is based on `__experimentalBorder.radius`. Unfortunately, it doesn't accept `default` value :/ so we need to style it upon inserting.

But I removed the spacing from on sale. I'm not sure why was it here. If needed merchants can add spacing through Inspector Controls: https://github.com/woocommerce/woocommerce/pull/57386/commits/19a7131ccb435d648c1aabee5d64b93c0cf6a4b3

Reply:
Tested your latest commit again, and this PR still looks good to me.